packages/client/src/l2.write.ts

```diff
@@ -144,4 +144,41 @@ const _ = (async () => {
       console.error('error setting text: ', { err })
     }
   }
+
+  // SET ADDRESS
+  try {
+    await client.simulateContract({
+      functionName: 'setAddr',
+      abi: l1Abi,
+      args: [
+        toHex(packetToBytes(publicAddress)),
+        '0x04270c4366010A52192bC8D3E29d9f0E21bBe969',
```

Comment:
What is this address? Maybe it is better set it as a const with a proper name!

Reply:
just a random address from etherscan, nothing to see here